### ISSUE TITLE
ACL lists for systems in quill

### ISSUE THREAD

**Issue by indexzero** [opsmezzo/quill#21]
Users should be able to only publish specific packages based on a "maintainers" array similar to package.json

**Comment by indexzero** [comment]
This will require changes to conservatory as well

**Comment by indexzero** [comment]
Probably worth modeling this after `npm owner`:

```
npm ERR! Usage: 
npm ERR! npm owner add <username> <pkg>
npm ERR! npm owner rm <username> <pkg>
npm ERR! npm owner ls <pkg>
```

**Comment by indexzero** [comment]
Fixed in `v0.3.x` at 89a6930
